packages/spotx/src/CennzxSpot.ts

```diff
@@ -260,4 +262,28 @@ export class CennzxSpot {
 
         return _fn;
     }
+
+    /**
+     * Query liquidity withdrawn for a core asset amount
+     * @param assetId The id of the asset
+     * @param coreAmount - the amount of core asset
+     */
+    get liquidityWithdrawn(): QueryableGetLiquidityWithdrawn {
```

Comment:
in which case user might need this query?
the input of removeLiquidity is user's liquidity not coreAsset amount

Reply:
removed